lib/AbstractMethodError.js

```diff
@@ -23,7 +23,9 @@ function Message() {
 	this.stack = undefined;
 	Error.captureStackTrace(this);
 	/** @type {RegExpMatchArray} */
-	const match = this.stack.split("\n")[3].match(CURRENT_METHOD_REGEXP);
+	const match = /** @type {*} */ (this.stack)
```

Comment:
TS 4.0 uses control flow now, which can't prove that `this.stack` is initialised, because `Error.captureStackTrace` has no way to declare its side effects.

So I cast it to any.